CMakelists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.2)
+project(cppfront)
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY 					${CMAKE_BINARY_DIR}/bin)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY 					${CMAKE_BINARY_DIR}/bin)
```

Comment:
This is super unfriendly to FetchContent users and totally unnecessary.

Reply:
I am using MSVC cmd line. And the cmd line tool has a tendency to make to obfuscate the build dir imo thats why I added that.